lib/Doctrine/DBAL/Platforms/MySqlPlatform.php

```diff
@@ -1049,7 +1053,8 @@ public function getReadLockSQL()
     protected function initializeDoctrineTypeMappings()
     {
         $this->doctrineTypeMapping = [
-            'tinyint'       => 'boolean',
+            'bit'           => 'boolean',
+            'tinyint'       => 'tinyint',
```

Comment:
Please provide some info on how it's expected to work with an already deployed schema.

Reply:
I updated the PR to keep BOOLEAN (so TINYINT) and to use comment hint instead. On an already deployed schema, reverse engineering will break strict comparison (`=== true`).